### ISSUE TITLE
Publish Storybook to Chromatic for UX

### ISSUE THREAD

**Issue by katherinejensen00** [paranext/paranext-core#113]
UX needs Storybook to be published to UX so they can use it in Figma

- [ ] Get Storybook to display something when it is run independently
- [ ] Publish Storybook as a static web app
- [ ] Set up Chromatic and publish the web app to it
- [ ] Work with UX to use the Chromatic version to work inside of Figma

https://storybook.js.org/docs/react/sharing/publish-storybook
https://help.figma.com/hc/en-us/articles/360045003494-Storybook-and-Figma

depends on #47

**Comment by katherinejensen00** [comment]
I think this task is no longer relevant given what we learned about Figma Storybook integration so I am going to close it as Won't Fix/Done for now.

**Comment by katherinejensen00** [comment]
I think this task is no longer relevant given what we learned about Figma Storybook integration so I am going to close it as Won't Fix/Done for now.
